src/components/calcite-panel/calcite-panel.tsx

```diff
@@ -54,6 +54,11 @@ export class CalcitePanel {
    */
   @Prop({ reflect: true }) dismissible = false;
 
+  /**
+   * Specifies the maxiumum height of the panel.
+   */
+  @Prop({ reflect: true }) heightScale: "s" | "m" | "l";
```

Comment:
I think this may need to be mutable so that users can update the size if/when content changes.